main.py

```diff
@@ -397,9 +400,9 @@ def update_progress(self, value):
         self.progress_label.config(text=f"{value}%")
 
     def start_conversion(self):
-        gm_path, gm_platform, godot_path = self.setup_ui.entries['gamemaker'].get(), self.gm_platform_settings, self.setup_ui.entries['godot'].get()
```

Comment:
It probably shouldn't be in a different context, but the key names are also used to create text on buttons and headings (specifically line 60 and 66 of /src/gui/setupui-py). This could, and probably shoud, be revised so that these ui elements are independent from the key.